plugin.video.mmlive/resources/language/English/strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
can you change those to the .po format and use the new layout for the language folder? See: https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language